examples/gcoap/gcoap_cli.c

```diff
@@ -222,6 +222,7 @@ static size_t _send(uint8_t *buf, size_t len, char *addr_str, char *port_str)
 
     remote.family = AF_INET6;
 
+#ifdef MODULE_GNRC_SOCK_UDP
```

Comment:
```suggestion
    if (IS_USED(MODULE_GNRC_SOCK_UDP)) {
```

See https://github.com/RIOT-OS/RIOT/blob/master/CODING_CONVENTIONS.md#general